data/merchant intro.txt

```diff
@@ -116,23 +116,23 @@ mission "Merchant Start 4"
 				`	What's the mission?"
 					goto point
 			label pleasantries
-			`	He smiles, "Politeness can bring you places. There aren't many of your type in this galaxy."`
+			`	He smiles. "Politeness can bring you places. There aren't many of your type in this galaxy."`
 				goto mission
 			label point
-			`	He smiles, "Straight to the point are we?"
+			`	He smiles, "Straight to the point, are we?"
 			label mission
 			`	"Well, anyways, <destination> is in desperate need of <cargo> and doctors. You see, they are facing an epidemic there which started a week ago. Willing captains have been asked to go there as fast as you can or at the very least, before <date>." He winces, "The death toll is currently at 50,000 and rising. So I ask you, accept this job and get the supplies there as soon as possible."
 			choice
 				`	"I'll do it."`
 					goto thanks
 				`	"Sorry, I can't."`
-			`	"I might be wrong about you," he says, disappointment on his face as he leaves.`
+			`	"I guess I might be wrong about you," he says, disappointment on his face as he leaves.`
```

Comment:
IMO I still think this should be "I guess I _was_ wrong about you."

Anyone else have thoughts on this?

Reply:
Yeah, "might" implies he could change his mind, but the rest of the text makes it seem final.